Лаб. работа 1/задача1.py

```diff
@@ -0,0 +1,5 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+new_numbers = numbers[:4]+numbers[5:]
+None_num = sum(new_numbers)/19
```

Comment:
В случае когда мы хотим разделить на общее количество элементов в списке, мы можем воспользоваться методом `len()` который возвращает нам количество элементов в списке.
```suggestion
none_num = sum(new_numbers) / len(numbers)
```
Также не стоит называть переменные с заглавной буквы. Так как в Python нельзя разделить переменные на константы и изменяемые переменные, есть общая договоренность - просто переменные пишутся с маленькой буквы (`regular_value`), а константы полностью пишутся заглавными (`CONST_VALUE`)